backend/pkg/models/database/fhir_patient_test.go

```diff
@@ -50,6 +51,37 @@ func TestFhirPatient_ExtractSearchParameters(t *testing.T) {
 		"534 Erewhon St PleasantVille Vic 3999",
 	}, testAddress)
 
+	var testTelecom []interface{}
+	err = json.Unmarshal(json.RawMessage(patientModel.Telecom), &testTelecom)
+	require.NoError(t, err)
+	telecom_1 := map[string]interface{}{
+		"code": "(03) 5555 6473",
+		"system": "phone",
+	}
+	telecom_2 := map[string]interface{}{
+		"code": "(03) 3410 5613",
+		"system": "phone",
+	}
+	telecom_3 := map[string]interface{}{
+		"code":"(03) 5555 8834",
+		"system": "phone",
+	}
+	require.Equal(t, []interface{}{
+		telecom_1,
+		telecom_2,
+		telecom_3,
+	}, testTelecom)
```

Comment:
I'd move these inline, rather than defining temporary variables. Mostly because these files are going to get long, and removing even 3 additional lines per test will add up in the long run. 

```suggestion
	require.Equal(t, []interface{}{
		map[string]interface{}{
			"code": "(03) 5555 6473",
			"system": "phone",
		},
		map[string]interface{}{
			"code": "(03) 3410 5613",
			"system": "phone",
		},
		map[string]interface{}{
			"code":"(03) 5555 8834",
			"system": "phone",
		},
	}, testTelecom)
```